### ISSUE TITLE
Unable to set property 'Closing' on object of type 'Blazorise.Bootstrap5.Modal'

### ISSUE THREAD

**Issue by buaziz** [Megabit/Blazorise#3158]
Using Blazorise.Bootstrap5
on Blazor Wasm, and setting ConfirmationMessage on EntityAction
on button click on the grid I get the below error

"Unhandled exception rendering component: Unable to set property 'Closing' on object of type 'Blazorise.Bootstrap5.Modal'. The error was: Specified cast is not valid."

and "Delegate to an instance method cannot have null 'this' (Parameter 'this')"

**Comment by stsrki** [comment]
Are you using the ABP framework? If I remember, they have `EntityAction`.

Based on the error it seems not all projects are upgraded to the latest 0.9.5 release but are left to one of the previews. Can you check your project dependencies, and also ABP modules to confirm?

**Comment by buaziz** [comment]
> Are you using the ABP framework? If I remember, they have `EntityAction`.
> 
> Based on the error it seems not all projects are upgraded to the latest 0.9.5 release but are left to one of the previews. Can you check your project dependencies, and also ABP modules to confirm?


Yes, am Using ABP 5.0.0.rc-1 and using

```
    <PackageReference Include="Blazorise.Bootstrap5" Version="0.9.5" />
    <PackageReference Include="Blazorise.Icons.FontAwesome" Version="0.9.5" />
    <PackageReference Include="Blazorise.Components" Version="0.9.5" />
```

with standard GRID code
```

  <EntityAction TItem="EmployeeWithNavigationPropertiesDto"
                                              Visible="@(CanDeleteEmployee)"
                                              Clicked="() => DeleteEmployeeAsync(context)"
                                              ConfirmationMessage="@(()=> L["Notification:ConfirmationMessageDelete"])"
                                              Text="@L["Delete"]"></EntityAction>
```

**Comment by stsrki** [comment]
I searched on ABP issues and it seems there was the same issue that was fixed https://github.com/abpframework/abp/issues/10653

But I'm not sure if they have released the fix yet.

**Comment by buaziz** [comment]
> I searched on ABP issues and it seems there was the same issue that was fixed [abpframework/abp#10653](https://github.com/abpframework/abp/issues/10653)
> 
> But I'm not sure if they have released the fix yet.

am using rc1 and this issue is labeled "5.0 final" , so i guess not released yet.
will wait for the release . Thanks :)
